src/routes/Admin/AdminView.js

```diff
@@ -29,7 +29,7 @@ class Admin extends Component {
     this.handleCloseModalQuick = this.handleCloseModalQuick.bind(this);
     this.deleteItem = this.deleteItem.bind(this);
     this.createAddItemGlobal = this.createAddItemGlobal.bind(this);
-    this.handleCreate = this.handleCreate.bind(this);
+    this.modalCreate = this.modalCreate.bind(this);
```

Comment:
modalCreate describes nothing to me. maybe a better name is `handleProductCreate`?